sql/src/main/java/io/crate/analyze/symbol/Symbols.java

```diff
@@ -89,6 +90,13 @@ public static boolean containsColumn(Iterable<? extends Symbol> symbols, ColumnI
         return false;
     }
 
+    public static boolean allLiterals(@Nullable Symbol symbol) {
+        if (symbol == null) {
```

Comment:
Imho would be better to assert not null here for the symbol.
@mfussenegger ^^ what do you think?